.github/workflows/scripts/mesh_deploy.sh

```diff
@@ -0,0 +1,15 @@
+MESH=$1
+MESHERYCTL="/usr/local/bin/mesheryctl"
+echo "Checking if mesheryctl is installed"
+
+
+if [ -f "${MESHERYCTL}" ]; then
+    echo "Found mesheryctl, deploy $MESH"
+	mesheryctl mesh deploy $MESH
+   else
+      printf "Mesheryctl not found. \nInstalling...\n"
```

Comment:
Same

Reply:
@Aisuko  I'm afraid i don't understand what you mean here

Reply:
Hi @s1ntaxe770r, sorry for replaying late. I mean be careful the shell format. Some like the the indentation.